### DISCUSSION TITLE
Delete object group scene group but not from the scene itself

### DISCUSSION THREAD

**Discussion by mauku** [zauberzeug/nicegui#4015] [Ideas / Feature Requests]
Hello, i would like to remove objects from a scene.group() element , but not delete them from the scene itself . Currently i couldnt find any solution for this . Is this possible to achieve ?

**Comment by falkoschindler** [comment]
Hi @mauku,

This sounds like a job for Three.js' [`removeFromParent`](https://threejs.org/docs/index.html?q=obje#api/en/core/Object3D.removeFromParent). I just created PR #4018 to add this feature to NiceGUI.

**Reply by falkoschindler** [reply]
This feature has been released in version 2.7.0. ✔️
